setup.py

```diff
@@ -88,7 +88,7 @@ def run(self) -> None:  # pylint: disable=no-self-use
     "license": "MIT",
     "keywords": "raiden ethereum blockchain",
     "install_requires": requirements,
-    "setup_requires": _get_single_requirement(requirements, "py-solc"),
```

Comment:
We can remove `_get_single_requirement` function, now that it is not used, anymore.

Reply:
I'm actually not sure if the **entire** set of requirements is used for the verify-contracts functionality, but it must be at least a subset of it. If we don't care about eventually unnecessary requirements, the function is not useful anymore, yes.